src/utils/__tests__/mediumAvatar-test.js

```diff
@@ -0,0 +1,14 @@
+import mediumAvatar from '../mediumAvatar';
+
+describe('mediumAvatar', () => {
+  test('If avatarUrl does not contain IMAGE_NAME.png returns avatarUrl as is ', () => {
+    const url = '/avatar/IMAGE_NAME/';
+    expect(mediumAvatar(url)).toEqual(url);
+  });
```

Comment:
Try to follow Arrange-Act-Assert (Google it)
It is kinda invisible, but try to have each step on different line, and if long separated by an empty line, see the other tests.